src/pages/Home.tsx

```diff
@@ -1,17 +1,25 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import { Link } from 'react-router-dom';
 
-import { useSesamoid } from '~/hooks/wasm';
+// import { useP2Panda } from '~/hooks/wasm';
+import loadP2Panda from 'p2panda-js';
 
 const LogWindow = () => {
   const [message, setMessage] = useState('');
 
-  useSesamoid(({ KeyPair }) => {
+  // useP2Panda(({ KeyPair }) => {
+  //   const keyPair = new KeyPair();
+  //   setMessage(`${keyPair.publicKey()}, ${keyPair.privateKey()}`);
+  // });
+
+  useEffect(() => {
```

Comment:
I tried two methods to import `p2panda-js`, analogous to what works in [the p2panda-js test](https://github.com/p2panda/p2panda/blob/main/p2panda-js/src/index.test.ts) but both are failing import / wasm compilation.